scss/color-scheme.scss

```diff
@@ -6,15 +6,20 @@
 	approximation of the default theme (Quartz Light).
 */
 $accent: var(--color-accent, tokens.$color-accent-blue);
+$accent-navigation: var(--color-accent-navigation, tokens.$color-accent-blue);
 $background-1: var(--color-background-1, tokens.$color-off-white);
 $background-1-transparent: var(--color-background-1-transparent, rgba(255, 255, 255, 0.000001));
 $background-2: var(--color-background-2, tokens.$color-white);
 $background-3: var(--color-background-3, rgba(0, 0, 0, 0.15));
 $background-4: var(--color-background-4, tokens.$color-white);
+$background-navigation: var(--color-background-navigation, tokens.$color-off-white);
+$background-navigation-faint: var(--color-background-navigation-faint, tokens.$color-white);
```

Comment:
What does this correspond to in the design? I'm only suspicious because this looks like the first use I can find of "faint" to name a background color, and the default color here is full white while the "non-faint" background is an off-white.

Reply:
The "faint" background is used for the flyout boxes on the site nav. The flyout actually has a bit of transparency to make it lighter. Maybe "background-navigation-transparent" is a better name?

![image](https://user-images.githubusercontent.com/97528742/163623224-f2bf9397-d198-46b6-80bd-d61c8dbc90eb.png)

Btw I'm deploying this for now, and I can rename it later.

Reply:
i agree with transparent

Reply:
I think I'd have the same hesitation with transparent because the default color isn't transparent. It's not ideal to mention more specific site locations in a color scheme either but I think something like "background-navigation-flyout" would be more helpful to future readers.